politeiawww/proposals.go

```diff
@@ -1516,6 +1584,41 @@ func (p *politeiawww) setVoteStatusReply(v www.VoteStatusReply) {
 	p.voteStatuses[v.Token] = v
 }
 
+// voteSummaries retrieves the voting summary information for a set of
+// proposals.
+func (p *politeiawww) voteSummaries(tokens []string, bestBlock uint64) (map[string]www.VoteSummary, error) {
```

Comment:
It looks like the only time this is being called is in `updatePropPointersWithVoteSummary`.  Can you move this up above the `updatePropPointersWithVoteSummary` declaration to try and keep things organized?